components/ingest-service/pipeline/publisher/bulk_publisher.go

```diff
@@ -12,8 +13,11 @@ import (
 )
 
 func BuildBulkRunPublisher(client backend.Client, maxNumberOfBundledRunMsgs int) message.ChefRunPipe {
+	count := 0
 	return func(in <-chan message.ChefRun) <-chan message.ChefRun {
-		return bulkRunPublisherBundler(in, client, maxNumberOfBundledRunMsgs)
+		name := fmt.Sprintf("pub-%d", count)
+		count++
+		return bulkRunPublisherBundler(in, client, maxNumberOfBundledRunMsgs, name)
```

Comment:
Adding a different name for each publisher to be able to see in the logs how many publishers are being used.